### PR TITLE
feat: optimize notice channel

### PR DESCRIPTION
**Pull request by sherotree**
## What this PR does / why we need it:
optimize notice channel that interaction about confirm enable notification channel

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

## Does this PR need be patched to older version?
❎ No

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/modules/org/pages/setting/notice-channel.tsx [hunk 1]
```diff
@@ -132,15 +148,52 @@ const NotifyChannel = () => {
...
+  }: {
+    status: NOTIFY_CHANNEL.ChannelEnableStatus;
+    channel: NOTIFY_CHANNEL.NotifyChannel;
+  }) => {
+    const { hasEnable, enableChannelName } = status || {};
+    confirm({
+      title: hasEnable
+        ? i18n.t('Are you sure you want to switch notification channels ?')
+        : i18n.t('Are you sure you want to enable the notification channel ?'),
+      content: hasEnable
```
Comment (laojun) [inline shell/app/modules/org/pages/setting/notice-channel.tsx:177]: better replace with one condition
```
const [title, content] = hasEnable ? [t1,c1] : [t2: c2];
confirm({
     title,
     content
})
```

File: shell/app/modules/org/pages/setting/notice-channel.tsx [hunk 2]
```diff
@@ -263,19 +316,29 @@ const NotifyChannel = () => {
...
+                  'You have a notification message from the Erda platform',
+                )}: $\{content}, ${i18n.t('please deal with it promptly')}`}</span>
+                <span
+                  className="text-primary cursor-pointer underline ml-2 jump-to-aliyun"
+                  data-clipboard-text={`${i18n.t(
+                    'You have a notification message from the Erda platform',
+                  )}: $\{content}, ${i18n.t('please deal with it promptly')}`}
+                  onClick={() =>
+                    window.open(
+                      'https://account.aliyun.com/login/login.htm?oauth_callback=https%3A%2F%2Fdysms.console.aliyun.com%2Fdysms.htm%23%2Fdomestic%2Ftext%2Ftemplate%2Fadd',
```
Comment (laojun) [inline shell/app/modules/org/pages/setting/notice-channel.tsx:334]: import the url from `constants.ts`

File: shell/app/modules/org/pages/setting/notice-channel.tsx [hunk 3]
```diff
@@ -263,19 +316,29 @@ const NotifyChannel = () => {
...
+                    )
+                  }
+                >
+                  {i18n.t('copy and jump to the application page')}
+                </span>
+                <Copy selector=".jump-to-aliyun" />
```
Comment (laojun) [inline shell/app/modules/org/pages/setting/notice-channel.tsx:340]: click tigger copy and jump？

Comment (sherotree) [reply shell/app/modules/org/pages/setting/notice-channel.tsx:340]: image


Designer and back-end demand the interactive, copy the template and jump to aliyun and paste it in some place

**Pull request merged**